common/config/debug-config.example.js

```diff
@@ -20,4 +20,5 @@ export default {
 	autoEndTurn: false,
 	logAttackHistory: false,
 	disableRewardCards: false,
-}
+	showCardsAsHTML: false,
```

Comment:
I don't think this name makes a lot of sense

Reply:
What would you make it?

Reply:
How about `dynamicCardRenders`?

Reply:
i like `renderCardsDynamically`

Reply:
That sounds good